__tests__/app.test.js

```diff
@@ -523,3 +523,41 @@ test("400: bad request made to misspelt articles url", () => {
   })
 })
 })
+describe.only("GET /api/articles/:article_id comment count", () => {
```

Comment:
As this is here I cannot check this work as I don't know if your other tests are working.